test/e2e/storage/volume_metrics.go

```diff
@@ -262,18 +266,12 @@ var _ = utils.SIGDescribe("[Serial] Volume metrics", func() {
 		})
 
 		It("should create none metrics for pvc controller before creating any PV or PVC", func() {
-			if !metricsGrabber.HasRegisteredMaster() {
-				framework.Skipf("Environment does not support getting controller-manager metrics - skipping")
-			}
```

Comment:
Is this code deleted because it doesn't matter if metrics is registered when none are being created?